examples/binaries/calc-hash/aggregator/src/code.rs

```diff
@@ -0,0 +1,59 @@
+use gstd::{exec, msg, ActorId, Decode, MessageId, Vec};
+use shared::{Method, Package};
+
+#[no_mangle]
+pub unsafe extern "C" fn init() {
+    (state::THRESHOLD, state::CALCULATOR) =
+        msg::load::<(u64, ActorId)>().expect("invalid calculator address");
```

Comment:
```suggestion
        msg::load().expect("invalid calculator address");
```
can't we avoid type specification?